pkg/internal/checkup/executor/executor.go

```diff
@@ -114,42 +111,9 @@ func (e Executor) Execute(ctx context.Context, vmiName, podName, podContainerNam
 		return status.Results{}, err
 	}
 
-	log.Printf("Clearing Trex console stats before test...")
-	_, err := trexClient.ClearStats(ctx)
-	if err != nil {
-		return status.Results{}, fmt.Errorf("failed to clear trex stats on pod \"%s/%s\" side: %w", e.namespace, podName, err)
-	}
-
-	const (
-		trafficSourcePort = 0
-		trafficDestPort   = 1
-	)
-
-	log.Printf("Running traffic for %s...", e.testDuration.String())
-	_, err = trexClient.StartTraffic(ctx, e.trafficGeneratorPacketsPerSecond, trafficSourcePort, e.testDuration)
-	if err != nil {
-		return status.Results{}, fmt.Errorf("failed to run traffic from trex-console on pod \"%s/%s\" side: %w",
-			e.namespace, podName, err)
-	}
-
 	results := status.Results{}
-	trafficGeneratorMaxDropRate, err := trexClient.MonitorDropRates(ctx, e.testDuration)
-	log.Printf("traffic Generator Max Drop Rate: %fBps", trafficGeneratorMaxDropRate)
-	if err != nil {
-		return status.Results{}, err
-	}
-
 	var trafficGeneratorSrcPortStats trex.PortStats
```

Comment:
Is this variable currently needed?
It saves the `trex` package from removal.

Reply:
I kept it intentionally as a boilerplate, so that we could fill this with actual data after the trex is alive again.
it's also intentional that  `types.go` stays, cause the structs in it are going to be the same so no point in removing+bringing it back again..